src/main/resources/config/liquibase/changelog/20251012122800_changelog.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.20.xsd">
+
+    <changeSet id="20250818094500" author="Alphar">
+        <createTable tableName="exercise_version">
+            <!-- Common fields -->
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="author_id" type="bigint">
```

Comment:
can we add <constraints nullable="false" /> here as well? The table `submission_version` also specifies `author_id` to be non nullable